dotcom-rendering/tsconfig.json

```diff
@@ -16,6 +16,7 @@
 		},
 		"preserveConstEnums": true
 	},
+	"include": ["**/*", ".storybook/mocks/bridgetApi.ts"],
```

Comment:
This helps catch compile errors in the bridget mock, so we ensure stories relying on it build adequately!